src/Umbraco.Tests.Integration/Services/ContentServiceEventTests.cs

```diff
@@ -7,24 +7,29 @@
 using Umbraco.Core.Persistence.Repositories.Implement;
 using Umbraco.Core.Services;
 using Umbraco.Core.Services.Implement;
-using Umbraco.Tests.Common.Builders;
+using Umbraco.Tests.Integration.Testing;
 using Umbraco.Tests.TestHelpers.Entities;
 using Umbraco.Tests.Testing;
 
-namespace Umbraco.Tests.Services
+namespace Umbraco.Tests.Integration.Services
 {
     [TestFixture]
     [UmbracoTest(Database = UmbracoTestOptions.Database.NewSchemaPerTest,
         PublishedRepositoryEvents = true,
         WithApplication = true,
         Logger = UmbracoTestOptions.Logger.Console)]
-    public class ContentServiceEventTests : TestWithSomeContentBase
+    public class ContentServiceEventTests : UmbracoIntegrationTest
     {
+        private IContentTypeService _contentTypeService => GetRequiredService<IContentTypeService>();
+        private IContentService _contentService => GetRequiredService<IContentService>();
+        private ILocalizationService _localizationService => GetRequiredService<ILocalizationService>();
+        private IFileService _fileService => GetRequiredService<IFileService>();
+
         private GlobalSettings _globalSettings;
 
-        public override void SetUp()
+        public override void Setup()
         {
-            base.SetUp();
+            base.Setup();
             ContentRepositoryBase.ThrowOnWarning = true;
             _globalSettings = new GlobalSettings();
         }
```

Comment:
IMO it is nicer to use [Setup] and call the mehtod something else to not collide with name from the base class. Then we donøt need to remember to call `base.Setup();`

Reply:
Have updated to this.